packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -312,11 +261,14 @@ export default class extends EntityModel {
   }
 
   getEmails(): string[] {
-    return (this.properties as any)[emailBaseUri];
+    return (this.properties as any)[WORKSPACE_TYPES.propertyType.email.baseUri];
   }
 
-  getShortname(): string | undefined {
-    return (this.properties as any)[AccountFields.shortnameBaseUri];
+  /** @todo - How can this be undefined? Have removed the undefined argument but not sure why it was there */
```

Comment:
Since writing it I've come to learn that shortnames can be missing during User creation? I think we need to revisit this, as shortnames should definitely be an invariant outside of the creation flow. We shouldn't have to deal with conditionals all over the place and possibly have one being missing.

Reply:
Thanks, that seems good!